make_count/make_count.tex

```diff
@@ -0,0 +1,293 @@
+
+Combining the assumptions that $\texttt{inf\_cast}(\din, \texttt{TO}) \leq \dout$ and that \texttt{v}, \texttt{w} are $\din$-close under the symmetric distance metric means that
+\begin{equation}
+    d_\text{Sym}(\texttt{u}, \texttt{v})
+    \leq \din \leq \dout.
+\end{equation}
+
+Let $\mathcal{X}$ be the domain of all elements of type \texttt{TIA}. Therefore, we see that the symmetric distance  between \texttt{u} and \texttt{v} is 
+\begin{equation}
+\label{eq:expand-dymm-dist}
+    d_\text{Sym}(\texttt{u}, \texttt{v}) = \sum_{z\in \mathcal{X}} |m_{\texttt{u}}(z) - m_{\texttt{v}}(z)| \leq \din \leq \dout.
+\end{equation}
+
+The $\function$ used in \texttt{MakeCount} sums over a single data type, namely a row. Let \texttt{rows} be a one-element domain, where every element of type \texttt{TIA} is considered to be the same element of \texttt{rows}; and let the single element be called \texttt{row}. Also, as in the \href{https://www.overleaf.com/project/60d215bf90b337ac02200a99}{Pseudocode definitions} document, let \texttt{len(vec)} be a function that returns the number of rows in vector \texttt{vec}.
+
+Therefore, using the notation in definition \ref{defn:symm-dist}, we can write
+\begin{equation}
+\label{eq:len-to-m}
+    |\texttt{len(u)} - \texttt{len(v)}| = \sum_{z\in \texttt{rows}}|m_{\texttt{u}}(z) - m_{\texttt{v}}(z)| = |m_{\texttt{u}}(\texttt{row}) - m_{\texttt{v}}(\texttt{row})|
+\end{equation}
+(note that the summation term is removed in the final term in equation \ref{eq:len-to-m} since the domain \texttt{rows} consists of the single element \texttt{row}).
+
+By the triangle inequality, then, we see that
+\begin{equation}
+\label{eq:triang-ineq}
+    |m_{\texttt{u}}(\texttt{row}) - m_{\texttt{v}}(\texttt{row})| \leq \sum_{z\in \mathcal{X}} |m_{\texttt{u}}(z) - m_{\texttt{v}}(z)|.
+\end{equation}
+
+Combining equations \ref{eq:len-to-m} and \ref{eq:triang-ineq} tells us that $|\texttt{len(u)} - \texttt{len(v)}| = |m_{\texttt{u}}(\texttt{row}) - m_{\texttt{v}}(\texttt{row})| \leq \sum_{z\in \mathcal{X}} |m_{\texttt{u}}(z) - m_{\texttt{v}}(z)|$; combining this with equation \ref{eq:expand-dymm-dist} tells us that we have
+\begin{equation}
+\label{eq:lens-dout-close}
+    |\texttt{len(u)} - \texttt{len(v)}| = |m_{\texttt{u}}(\texttt{row}) - m_{\texttt{v}}(\texttt{row})| \leq \dout,
+\end{equation}
+so \texttt{len(u)} and \texttt{len(v)} must be $\dout$-close. This, however, does not complete the proof because $\texttt{function(u)}$ does not return \texttt{len(u)}, but either \texttt{exact\_cast(len(u),TO)} or -- in the event \texttt{exact\_cast} fails -- \texttt{get\_max\_consecutive\_int(TO)}.
+
+We now consider the two cases that could occur:

```

Comment:
Nicely done!  You might be able to make the cases a bit more concise using a series of inequalities in an eqnarray, but the proofs are also good as is.